crates/ruff_cli/src/lib.rs

```diff
@@ -101,6 +101,18 @@ fn is_stdin(files: &[PathBuf], stdin_filename: Option<&Path>) -> bool {
     file == Path::new("-")
 }
 
+/// Get the actual value of the `format` desired from either `output_format`
+/// or `format`, and warn the user if they're using the deprecated form.
+fn warn_about_deprecated_help_format(
+    output_format: HelpFormat,
+    format: Option<HelpFormat>,
+) -> HelpFormat {
+    if format.is_some() {
+        warn_user!("The argument `--format=<FORMAT>` is deprecated. Use `--output-format=<FORMAT>` instead.");
+    }
+    format.unwrap_or(output_format)
```

Comment:
I guess this doesn't matter since clap will error if you provide both? but I'd expect the `output_format` to take precedence.

Reply:
It doesn't, really, but only one of these is an `Option` type so I don't see a way to do it the other way around :)

Reply:
👍 sounds good thanks!